app/assets/stylesheets/common/base/new-user.scss

```diff
@@ -1,81 +1,87 @@
-.new-user-wrapper {
-  .user-navigation {
-    --user-navigation__border-width: 4px;
-    border-bottom: 1px solid var(--primary-low);
+@mixin bordered-horizontal-nav($border-width) {
```

Comment:
I instinctively like this. Seems like its a good step towards more consistent UI. 👍